google/cloud/aiplatform/prediction/local_endpoint.py

```diff
@@ -105,12 +108,32 @@ def __init__(
             host_port (str):
                 Optional. The port on the host that the port, AIP_HTTP_PORT, inside the container
                 will be exposed as. If it's unset, a random host port will be assigned.
+            gpu_count (int):
+                Optional. Number of devices to request. Set to -1 to request all available devices.
+                To use GPU, set either `gpu_count` or `gpu_device_ids`.
+                The default value is -1 if gpu_capabilities is set but both of gpu_count and
+                gpu_device_ids are not set.
+            gpu_device_ids (List[str]):
+                Optional. This parameter corresponds to `NVIDIA_VISIBLE_DEVICES` in the NVIDIA
+                Runtime.
+                To use GPU, set either `gpu_count` or `gpu_device_ids`.
+            gpu_capabilities (List[List[str]]):
+                Optional. This parameter corresponds to `NVIDIA_DRIVER_CAPABILITIES` in the NVIDIA
+                Runtime. This must be set to use GPU. The outer list acts like an OR, and each
```

Comment:
"This must be set to use GPU." is no longer true, right? We're setting a default value for this if the user just provides `gpu_count` or `gpu_device_ids`.